src/components/RecentDatasets.tsx

```diff
@@ -0,0 +1,82 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+// SPDX-License-Identifier: Apache-2.0
+
+import Link from "next/link";
+import { SearchedDataset } from "@/services/discovery/types/dataset.types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowRight } from "@fortawesome/free-solid-svg-icons";
+import { formatDate } from "@/utils/formatDate";
+import { useMemo } from "react";
+
+type DatasetLinkProps = Pick<
+  SearchedDataset,
+  "title" | "createdAt" | "description"
+>;
+
+const RecentDatasets = ({ datasets }: { datasets: SearchedDataset[] }) => {
+  const sortedDatasets = useMemo(() => {
+    return [...datasets]
+      .sort(
```

Comment:
important: Please remove `.sort()`, it is responsbility of the client of this component to sort the entries.